spyderlib/widgets/explorer.py

```diff
@@ -1000,64 +1000,82 @@ class ExplorerWidget(QWidget):
     def __init__(self, parent=None, name_filters=['*.py', '*.pyw'],
                  show_all=False, show_cd_only=None, show_icontext=True):
         QWidget.__init__(self, parent)
-        
+
+        # Widgets        
         self.treewidget = ExplorerTreeWidget(self, show_cd_only=show_cd_only)
-        self.treewidget.setup(name_filters=name_filters, show_all=show_all)
-        self.treewidget.chdir(getcwd())
-        
+        button_previous = QToolButton(self)
+        button_next = QToolButton(self)
+        button_parent = QToolButton(self)
+        self._button_menu = QToolButton(self)

```

Comment:
Well it is public to the methods of the class... it is not meant to be public to the rest of the world...